var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -112,6 +113,14 @@ class Esmf(MakefilePackage):
     depends_on("parallelio@2.5.10: ~mpi", when="@8.5:+external-parallelio~mpi")
     depends_on("cmake@3.5.2:", type="build", when="~external-parallelio")
 
+    # python library
+    with when("+python"):
+        extends("python")
+        depends_on("py-setuptools", type="build")
+        depends_on("py-wheel", type="build")
+        depends_on("py-mpi4py", when="+mpi")
+        depends_on("py-pip")
```

Comment:
Should numpy be added here since that is a dependency of ESMPy?

Reply:
Yup, absolutely correct. Thanks for catching that.